.changeset/young-beans-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/nextjs': patch
+---
+
+Improve experience when swapping keys on Keyless mode.
```

Comment:
💡 Should we mention here regarding the hot-swapped .env case? Just to clarify the "improve experience" part